game-core/src/main/java/games/strategy/triplea/ui/TripleAFrame.java

```diff
@@ -267,8 +268,12 @@ public void windowClosing(final WindowEvent e) {
     final Image small = uiContext.getMapImage().getSmallMapImage();
     smallView = new MapPanelSmallView(small, model, uiContext.getMapData());
     mapPanel = new MapPanel(data, smallView, uiContext, model, this::computeScrollSpeed);
+    tooltipManager = new MapUnitTooltipManager(mapPanel);
     mapPanel.addMapSelectionListener(mapSelectionListener);
-    final MouseOverUnitListener mouseOverUnitListener = (units, territory, me) -> unitsBeingMousedOver = units;
+    final MouseOverUnitListener mouseOverUnitListener = (units, territory, me) -> {
+      unitsBeingMousedOver = units;
+      tooltipManager.updateTooltip(getUnitInfo());
+    };
     mapPanel.addMouseOverUnitListener(mouseOverUnitListener);
```

Comment:
Would you mind inlining this Lambda expression with the next line?

Reply:
Done.